crates/anvil-polkadot/tests/it/demo_test.rs

```diff
@@ -0,0 +1,121 @@
+use crate::utils::TestNode;
```

Comment:
This file is just a demo.. it should be deleted in the first PR with real tests.